src/handler.js

```diff
@@ -101,6 +101,12 @@ async function batchFetch (blocks, context, logger) {
       }
       block.key = key
 
+      // Skip block cancel
+      if (block.cancel) {
+        telemetry.increaseLabelCount('bitswap-block', [TELEMETRY_TYPE_INFO, TELEMETRY_RESULT_CANCELED])
```

Comment:
Can we record the correct want type in the log as we were before?

```suggestion
        const type = block.wantType === Entry.WantType.Block
          ? TELEMETRY_TYPE_DATA
          : TELEMETRY_TYPE_INFO
        telemetry.increaseLabelCount('bitswap-block', [type, TELEMETRY_RESULT_CANCELED])
```

Reply:
good catch. Did not see that difference 🤦🏼‍♂️